package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medusa-extender",
-  "version": "1.7.7",
+  "version": "1.7.6",
```

Comment:
There is an issue here, it should not be changed

Reply:
oops my bad sorry